rtsf-at-checkout-cv-region-of-interest/src/enterexit.py

```diff
@@ -184,10 +187,13 @@ def create_pipelines():
         cameraConfiguration.append(jsonConfig)
 
         # Delete existing frame_store and then re-create it writeable by all
-        print("Setting up frame store in %s" % camFrameStore)
-        if os.path.isdir(camFrameStore):
-            shutil.rmtree(camFrameStore)
-        os.mkdir(camFrameStore, 0o777)
+        print("Setting up frame store in %s" % camFrameStore)  
+        if len(camFrameStore) > 0:
+            if os.path.isdir(camFrameStore):
+                shutil.rmtree(camFrameStore)
+            isExist = os.path.exists(camFrameStore)
+            if not isExist:
+                os.mkdir(camFrameStore, 0o777)
```

Comment:
i don't think we should check isExist any more. if that one is a file then we miss the chance to create directory actually.